app/src/main/java/org/mozilla/fenix/ext/View.kt

```diff
@@ -15,10 +15,7 @@ fun View.increaseTapArea(extraDps: Int) {
     parent.post {
         val touchRect = Rect()
         getHitRect(touchRect)
-        touchRect.top -= dips
-        touchRect.left -= dips
-        touchRect.right += dips
-        touchRect.bottom += dips
+        touchRect.inset(-dips, -dips)
```

Comment:
it looks like some of them are adding? am i misunderstanding what the inset fun is doing?